verilog/analysis/verilog_linter.cc

```diff
@@ -228,36 +247,54 @@ ViolationFixer::AnswerChoice ViolationFixer::InteractiveAnswerChooser(
       "P - show fixes applied in this file so far\n"
       "? - print this help and prompt again\n";
 
+  // If we have alternatives, show this in the short-menu.
+  std::string alternative_list;
+  if (size_t fix_count = violation.autofixes.size(); fix_count > 1) {
+    for (size_t i = 0; i < fix_count; ++i)
+      alternative_list += std::to_string(i + 1) + ",";
```

Comment:
https://abseil.io/tips/3 -- maybe pass through StrCat, instead of operator+?

Reply:
Fixed, using `absl::StrAppend()`